src/main/java/com/example/PlayerFactory.java

```diff
@@ -1,6 +1,11 @@
 package com.example;
 
-public interface PlayerFactory
-{
-    Player createPlayer();
+public class PlayerFactory {
+    public static Player createHumanPlayer(String name, char symbol) {
```

Comment:
There should be one method in the PlayerFactory which will be responsible for creating the players based on the user input with the help of switch case

Reply:
Please refer to things I have said about what thought I had to @NisargPipaliya about this.